app-api/src/main/java/org/openmrs/eip/app/logger/observer/EventManager.java

```diff
@@ -0,0 +1,36 @@
+package org.openmrs.eip.app.logger.observer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+
+public class EventManager {
```

Comment:
Why are these changes necessary?

Reply:
Please see [this comment](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/21#discussion_r1023580970) and share your opinion.

Reply:
Removed while refactoring the code.